src/components/pages/rules/flyouts/editRuleFlyout.js

```diff
@@ -11,7 +13,16 @@ export const EditRuleFlyout = ({ t, onClose, rule }) => (
       <Flyout.CloseBtn onClick={onClose} />
     </Flyout.Header>
     <Flyout.Content>
-      <RuleEditorContainer onClose={onClose} rule={rule} />
+      <Protected permission={permissions.updateRules}>{
+        (hasPermission, permission) => hasPermission
+          ? <RuleEditorContainer onClose={onClose} rule={rule} />
+          :
+          <div>
+            <ProtectedError t={t} permission={permission} />
+            <p>A read-only view will be added soon as part of another PBI.</p>
```

Comment:
String needs to be translated

Reply:
Yes, but as it indicates... its temporary... and so will go away with my next PR.